### DISCUSSION TITLE
mMicroG too big than one of TeamVanced?

### DISCUSSION THREAD

**Discussion by oldman20** [nikhilbadyal/docker-py-revanced#175] [General]
As can be seen in screenshots below:
![image](https://github.com/nikhilbadyal/docker-py-revanced/assets/39668756/975de9d8-2310-4fb2-9153-9b342c9fff37)

![image](https://github.com/nikhilbadyal/docker-py-revanced/assets/39668756/30afd693-92ae-4836-8836-1c94f5976c33)

How can optimize it during build automatic?

**Comment by nikhilbadyal** [comment]
We not modify/build microg. Just download it from source. In this case mmicrog of inotiaa00. See [here](https://github.com/inotia00/mMicroG/releases).
If you don't like mMicrog you can modify [this](https://github.com/nikhilbadyal/docker-py-revanced/blob/dd09a50951e7f7e569747ce3fd23ecd104846bd0/src/downloader.py#L246) to download [TeamVanced microg](https://github.com/inotia00/VancedMicroG/releases/tag/v0.2.27.230755) instead of mmicrog.

**Reply by oldman20** [reply]
thank so much for tip. Success!
